core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -147,15 +155,14 @@ class WorkspaceService(
   val terraBucketWriterRole: String,
   rawlsWorkspaceAclManager: RawlsWorkspaceAclManager,
   multiCloudWorkspaceAclManager: MultiCloudWorkspaceAclManager,
-  val fastPassServiceConstructor: (RawlsRequestContext, SlickDataSource) => FastPassService,
+  val fastPassServiceConstructor: RawlsRequestContext => FastPassService,
```

Comment:
The calls to FastPassService were being unnecessarily wrapped in transaction blocks. 
Since those were moved out, we can use the normal service constructor pattern.